### PR TITLE
Replace obj_shell reference in create event with object_index & various scaling fixes

### PR DESCRIPTION
**Pull request by FaultyFunctions**
- This will enable users to rename obj_shell to whatever they want and will prevent errors if they rename it.
- Fixed various errors in README and script example that I forgot to fix

Just a small PR

### CODE REVIEW

**Comment by FaultyFunctions** [comment]
Also not sure how to make sure users can rename `font_console` to whatever they want and update it programmatically. If you do:

```gml
var _shell = instance_create_layer(0, 0, "Controllers", Shell);
_shell.consoleFont = fnt_my_shell_font;
```

You get an error because it's trying to set `consoleFont` to `font_console` but doesn't find it in the instance variables on creation.

`Undefined variable 'font_console' referenced in Shell's variable 'consoleFont'`

Not sure how to fix this or work around it without setting the `consoleFont` variable to `noone` initially, or having the user update the actual variable inside the Variable Definitions.

**Comment by FaultyFunctions** [comment]
Sorry this PR turned into fixing various things that I noticed when transferring the new rt-shell into my project.

File: rt-shell/objects/obj_shell/Draw_64.gml [hunk 1]
```diff
@@ -204,6 +207,7 @@ if (isOpen) {
 		
 		draw_set_color(c_white);
 		draw_set_alpha(1);
+		draw_set_font(-1);
```
Comment (nickavv) [inline rt-shell/objects/obj_shell/Draw_64.gml:210]: what purpose does this serve?

File: rt-shell/objects/obj_shell/Step_0.gml [hunk 2]
```diff
@@ -178,3 +173,5 @@ if (!isOpen) {
 		autocompleteScrollPosition = 0;
 	}
 }
+
+show_debug_message(surface_get_height(scrollSurface));
```
Comment (nickavv) [inline rt-shell/objects/obj_shell/Step_0.gml:177]: remove debug output

**Review by nickavv — COMMENTED** [review]
Can you replace logic operators `and` and `or` with `&&` and `||` respectively? It's just my preferred style and I'd like to keep the repository consistent

File: rt-shell/objects/obj_shell/Draw_64.gml (hunk 1 above)
Comment (FaultyFunctions) [reply rt-shell/objects/obj_shell/Draw_64.gml:210]: See the second part of #20 

Basically resets the font back to GM's default after we are done using it.

**Comment by FaultyFunctions** [comment]
Okay everything *should* be ready to merge but let me testing this real quick on my live project.

File: rt-shell/objects/obj_shell/Draw_64.gml (hunk 1 above)
Comment (nickavv) [reply rt-shell/objects/obj_shell/Draw_64.gml:210]: Oh, I didn't even know GM had a default font! 😅

**Comment by nickavv** [comment]
I tested locally and it looks good, thanks!

**Pull request merged**